rhobot.go

```diff
@@ -57,6 +57,16 @@ func main() {
 		Value: "",
 		Usage: "yaml file containing email distribution list",
 	}
+	hcSchemaFlag := cli.StringFlag{
+		Name:  "hcschema",
```

Comment:
I forget, but is it possible to make these two commands dependent on the healthchecks command?  In otherwords, it would be cleaner to have them be just "schema" instead of "hcschema", but that would require them to be valid only when "healthchecks" is the command used.